src/main/java/seedu/address/model/animal/Task.java

```diff
@@ -0,0 +1,113 @@
+package seedu.address.model.animal;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Objects;
+
+/**
+ * Represents the Task addressed to the {@link Animal}
+ *
+ */
+public class Task {
+
+    public static final String MESSAGE_CONSRAINTS = "Description should not be empty!";
+    public static final String VALIDATION_REGEX = "^.*\\S.*$";
```

Comment:
Do explain what this Regex constraints are, because if the constraints are simply `Description should not be empty`, then wouldnt a `String::length > 0` check suffice? Also, typo in `MESSAGE_CONSRAINTS`.

Reply:
Noted. The constraints for the description were that it shouldn't be empty, and it shouldn't just contain whitespace characters.